### ISSUE TITLE
3D markers not shown in RStudio

### ISSUE THREAD

**Issue by bcdunbar** [ropensci/plotly#1055]
from community discussion https://community.plot.ly/t/no-markers-on-3d-plots/4125/5

Unique to RStudio Viewer on Windows. External browsers (IE, Edge) work as expected.

```r
p <- plot_ly(x = 1:10, y = 10:1, z = 1:10, type = 'scatter3d', mode = 'markers')
```

Error:
```
TypeError: undefined is not a constructor (evaluating 'ArrayBuffer.isView(t)')   
```

**Comment by cpsievert** [comment]
@bcdunbar can you replicate this issue on your system? If so, can you show me the list of `<script>`s in the `<head>`?

(you can inspect element in the viewer pane of RStudio to do so)

**Comment by bcdunbar** [comment]
Yeah I can replicate it RStudio viewer on Windows. Microsoft Edge and IE work fine.

<img width="814" alt="2017-06-26 1" src="https://user-images.githubusercontent.com/8393928/27548697-b326a148-5a67-11e7-9bc8-fdba44d72c7f.png">

**Comment by cpsievert** [comment]
@bcdunbar does this fix the problem?

```r
library(plotly)
library(htmlwidgets)

p <- plot_ly(x =1:10, y = 10:1, z = 1:10, type = 'scatter3d', mode = 'markers')
saveWidget(p, "index.html", selfcontained = FALSE)

# move <script plotly.js> from <head> to <body>
html <- readLines("index.html")
idx <- grep("plotly-latest", html)
plotlyjs <- html[[idx]]
html[[idx]] <- ""
bod <- grep("<body", html)
new_html <- c(html[seq(1, bod)], plotlyjs, html[seq(bod+1, length(html))])
writeLines(new_html, "index.html")

# should open in viewer
servr::httd()
```

**Comment by bcdunbar** [comment]
@cpsievert it doesn't. Still no markers visible in the viewer and produces the same error in the console.

**Comment by cpsievert** [comment]
Should be fixed via 90a5a0a3d042b087dabcaa3937c218bcc7b81548
